src/cmd/flux-R.c

```diff
@@ -562,7 +562,7 @@ int cmd_rerank (optparse_t *p, int argc, char **argv)
         log_err_exit ("Failed to transform R objects on stdin");
```

Comment:
Seem to be missing a end paren in commit message, probably goes after "rerank operation"?

```
(too many or too few hosts specified for the rerank operation, but the resulting error message from users
can therefore be vague and confusing, e.g. "rlist_rerank: No space left of device".
```

Reply:
Thanks! Fixed the commit message and pushed the result.